repo_id,repo_full_name,file,file_sha1,match
147528539,0x00-0x00/CVE-2018-15131,README.md,4e4de2609437aec0d525c69f6ced62609c6f54e7,CVE-2018-15131
147528539,0x00-0x00/CVE-2018-15131,cve-2018-15131-user-enum.py,9e2ee8a4f58378ffb5035618457639dcf76a5e81,CVE-2018-15131
147528539,0x00-0x00/CVE-2018-15131,_GITHUB_REPO_METADATA_,,CVE-2018-15131
